packages/form-core/src/registration/FormRegistrarMixin.js

```diff
@@ -38,6 +38,7 @@ const FormRegistrarMixinImplementation = superclass =>
          * it should act like an array (see ChoiceGroupMixin).
          * Usually, when false, we deal with a choice-group (radio-group, checkbox-group,
          * (multi)select)
+         * @protected
```

Comment:
static get prop so this @protected won't do much, annotate in initialisation instead.